src/components/components/ReinforcementStatement/ReinforcementStatement.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./ReinforcementStatement.scss";
+import ReinforcementImage from "./Assets/reinforcementImage.jpg";
```

Comment:
the assets folder should be called "assets". not "Assets"

Reply:
changed name to assets